### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *Global upgrade of the package.*
- *Add change_url script*
- *Update nginx.conf to allow bigger torrent uploads*

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : JimboJoe
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/transmission_ynh%20PR60/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by Gofannon** [comment]
bump

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+
+    client_max_body_size 8M;
 
     # Include SSOWAT user panel.
     include conf.d/yunohost_panel.conf.inc;
     more_clear_input_headers 'Accept-Encoding';
 }
 
-location __PATH__ {
+location __PATH__/ {
```
[the remark this answers is not in the document]
Comment (maniackcrudelis) [reply conf/nginx.conf]: Looks like we don't need it,
According to the tests, Trying to reach `domain.tld/path` leads to `domain.tld/path/transmission/web/`
And `domain.tld/path/` does as well.

Have a look to the test [`Installation in a sub path`](https://ci-apps-hq.yunohost.org/jenkins/job/transmission_ynh%20PR60/3/console)

[the remark this answers is not in the document]
Comment (alexAubin) [reply conf/nginx.conf]: Alrighty then, trusting you ¯\\\_(ツ)\_/¯

**Comment by kay0u** [comment]
Bump

**Comment by alexAubin** [comment]
Just ran the upgrade on my instance and everything went well ... [Let'seuh goh](https://ih1.redbubble.net/image.544771471.7940/raf,360x360,075,t,fafafa:ca443f4786.jpg)

**Comment by JimboJoe** [comment]
Can be merged in 3 days.

**Comment by alexAubin** [comment]
Anybody feel like clicking the button ? :wink:

**Pull request merged**